github/pulls.go

```diff
@@ -42,7 +42,8 @@ type PullRequest struct {
 	StatusesURL  *string    `json:"statuses_url,omitempty"`
 	DiffURL      *string    `json:"diff_url,omitempty"`
 	PatchURL     *string    `json:"patch_url,omitempty"`
-	Assignee     *User      `json:"assignee,omitempty"` // probably only in webhooks
+	Assignee     *User      `json:"assignee,omitempty"`
+	Assignees    []*User    `json:"assignees,omitempty"`
```

Comment:
AFAICT, `Assignees` does not belong on a `PullRequest` struct, but does belong on an `Issue` struct and is already there on [line 39 of `issues.go`](https://github.com/google/go-github/blob/master/github/issues.go#L39).

If you still believe it belongs here, can you please point me to the GitHub documentation where it is mentioned?  Thanks!